seeds/users_info.js

```diff
@@ -0,0 +1,15 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('credentials').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('credentials').insert([
+        {username: 'bobheper1', password: 'biallzabob'},
+        {username: 'ricardo', password: 'david'},
+        {username: 'Greg', password: 'mankiw'},
+        {username: 'orungatan', password: 'spatialmemory'},
+        {username: 'eric', password: 'elhricson'}
```

Comment:
Are you able to log in with these users? The passwords stored in the server should be hashed, so trying to log-in with these might not work, because you'll be hashing the client input, but the password in the database is in plaintext.

An idea; import `bcrypt` then:
```js
{username: 'bobheper1', password: bcrypt.hashSync('biallzabob', 14)},
```
That might work for seeding.

Reply:
Oh you're right I didn't test these I only tested the ones I passed in. thanks!